### PR TITLE
Chat admin

### PR DESCRIPTION
**Pull request by fmonbeig**
Hello everyone,

Here is some bug fixes :
- Leaderboard : Fully responsive
- Profile : Modal don't have previous status anymore
- Double auth : All bug are fixes and 2fa is fully operational
- Chat : 
  Modal password can close know
  Users can see their role in chat
  Owner can give Admin right and revoke it 
- General : 
  homogenization of the design Modal 
  GameSocket are know recreate after a refresh

### CODE REVIEW

File: srcs/client/src/Pages/Chat/Chat.tsx [hunk 1]
```diff
@@ -77,6 +77,7 @@ export default function Chat(props: any) {
       changeChannelPassword(currentChannel.id, newChannelPassword);
     } else {
       setChannelPassword(currentChannel.id, newChannelPassword);
+      handleClosePassworChannel();
```
Comment (twagger) [inline srcs/client/src/Pages/Chat/Chat.tsx:80]: There is a typo Passwor / Password but it is the case also in the function declaration so it is ok :)

File: srcs/client/src/Pages/Home/home.tsx [hunk 2]
```diff
@@ -66,6 +66,7 @@ export default function Home({ updateID, userID }: any) {
       })
       .then((responseObject) => {
         if (responseObject.message) {
+          console.log(responseObject.message);
```
Comment (twagger) [inline srcs/client/src/Pages/Home/home.tsx:69]: To remove before the end of the project

**Review by twagger — APPROVED** [review]
Very nice :)
All good for me

File: srcs/client/src/Pages/Chat/Members/UserInMembers.tsx [hunk 3]
```diff
@@ -69,6 +70,32 @@ export default function UserInMembers({
                       blockedUsers
                     )}`
                   )} */}
+                  {/* NEED NOUFEL'S FUNCTIONS HERE */}
```
Comment (jnougaro) [inline srcs/client/src/Pages/Chat/Members/UserInMembers.tsx:73]: Just need to remove the comment here :)

File: srcs/server/src/auth/serialization/sessionSerialization.ts [hunk 4]
```diff
@@ -13,6 +13,7 @@ export class Serialization extends PassportSerializer {
   }
 
   async deserializeUser(user: RequestUser, done: Function) {
+    console.log(`Trying to deserialize user ${JSON.stringify(user)}`)
```
Comment (jnougaro) [inline srcs/server/src/auth/serialization/sessionSerialization.ts:16]: console.log to be removed?

**Review by jnougaro — APPROVED** [review]
Great fixes thanks!
Ok for me 💯

**Pull request merged**